frost/src/spatial.rs

```diff
@@ -0,0 +1,132 @@
+use crate::{Error, FrostObs};
+use chrono::prelude::*;
+use rstar::RTree;
+use olympian::points::{CoordinateType, Points};
+use rove::data_switch::{SpatialCache, Timestamp};
+use rove::pb::util::GeoPoint;
+
+// have to repeat from series? 
+// or is it possible to import - function `crate::series::extract_obs` exists but is inaccessible
```

Comment:
The extract obs in series is meaningfully different. That gets only the first timeseries, you need all of them